### PR TITLE
OA Intake Home Page tabs and options for Submitter and Leadership Users

### PR DESCRIPTION
**Pull request by alenaminsk**
OA Intake Home Page tabs and options for Submitter and Leadership Users

### CODE REVIEW

File: src/test/java/ServiceNow/OA_Intake/Features/OAIntakeForm.feature [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Feature: Create OA Intake form
   Description: This feature file contains scenarios relating to OA Intake form
 
-  @selenium @OAIntake-576 @OAIntake-544 @OAIntake-575 @OAIntake-499 @OAIntake-657 @OAIntake-430 @OAIntake-408 @OAIntake-623 @Alena @Smoke
+  @selenium @OAIntake-576 @OAIntake-544 @OAIntake-575 @OAIntake-499 @OAIntake-657 @OAIntake-430 @OAIntake-408 @OAIntake-623 @Alena
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/OA_Intake/Features/OAIntakeForm.feature:4]: This scenario is missing @Regression and it should be tagged @Smoke if needed

File: src/test/java/ServiceNow/OA_Intake/Features/OAIntakeForm.feature [hunk 2]
```diff
@@ -32,3 +32,56 @@ Feature: Create OA Intake form
     Then Submitter User can verify a submission confirmation modal OA Intake form appears
     And Submitter User chooses to submit a request
     Then Submitter User can confirm they are redirected to Requestor (Requested For) Queue for Negotiated Contracts page
+
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/OA_Intake/Features/OAIntakeForm.feature:35]: Please remove the empty line 35

File: src/test/java/ServiceNow/OA_Intake/Features/OAIntakeForm.feature [hunk 3]
```diff
@@ -32,3 +32,56 @@ Feature: Create OA Intake form
...
+
+  @selenium @Alena @Smoke @OAIntake-8 @OAIntake-417 @OAIntake-425 @OAIntake-426 @OAIntake-439 @OAIntake-260 @OAIntake-261 @OAIntake-263
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/OA_Intake/Features/OAIntakeForm.feature:37]: This scenario is missing @Regression

File: src/test/java/ServiceNow/OA_Intake/Features/OAIntakeForm.feature [hunk 4]
```diff
@@ -32,3 +32,56 @@ Feature: Create OA Intake form
...
+
+
+
+
+
+
+
+
+
+
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/OA_Intake/Features/OAIntakeForm.feature:87]: Please delete empty lines 74-87

File: src/test/java/ServiceNow/OA_Intake/Steps/OAIntakeSteps.java [hunk 5]
```diff
@@ -273,6 +273,165 @@ public void submitter_user_can_confirm_they_are_redirected_to_requestor_requeste
...
+        oaIntakeStepsImplementation.oaIntakeLeadershipUserLogin();
+    }
+
+    /**
+     * User verifies the Home page header
+     */
+    @Then("User can verify the Home page header")
+    public void user_can_verify_the_home_page_header() {
+        CommonUtils.waitForVisibility(oaIntakePage.pageHeaderOAIntake);
+        Assert.assertTrue(oaIntakePage.pageHeaderOAIntake.isDisplayed());
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/OA_Intake/Steps/OAIntakeSteps.java:293]: You could replace Assert with the one we have in the CommonUtils class and use it across this class. CommonUtils.assertTrue(). 

If you decided replacing it, then remove the testNG import as well.

File: src/test/java/ServiceNow/OA_Intake/Steps/OAIntakeSteps.java
Comment (hamidnesar) [inline src/test/java/ServiceNow/OA_Intake/Steps/OAIntakeSteps.java:1]: You could replace Assert with the one we have in the CommonUtils and use it across this class. CommonUtils.assertTrue().

If you decided replacing it, then remove the testNG import as well.

File: src/test/java/ServiceNow/OA_Intake/StepsImplementation/OAIntakeStepsImplementation.java [hunk 6]
```diff
@@ -5,7 +5,10 @@
 import appsCommon.Utils.ServiceNow_Common_Methods;
 import appsCommon.Utils.ServiceNow_Login_Methods;
 import com.nci.automation.utils.CucumberLogUtils;
+import com.nci.automation.web.CommonUtils;
 import com.nci.automation.web.WebDriverUtils;
+import org.testng.Assert;
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/OA_Intake/StepsImplementation/OAIntakeStepsImplementation.java:10]: You could replace Assert with the one we have in the CommonUtils class and use it across this class. CommonUtils.assertTrue().

If you decided replacing it, then remove the testNG import as well.

**Review by hamidnesar — CHANGES_REQUESTED** [review]
Hi, 
Please review the comments. Thanks!

File: src/test/java/ServiceNow/OA_Intake/StepsImplementation/OAIntakeStepsImplementation.java
Comment (katm007) [inline src/test/java/ServiceNow/OA_Intake/StepsImplementation/OAIntakeStepsImplementation.java:1]: Please remove the empty line11 between imports

File: src/test/java/ServiceNow/OA_Intake/Features/OAIntakeForm.feature (hunk 1 above)
Comment (alenaminsk) [reply src/test/java/ServiceNow/OA_Intake/Features/OAIntakeForm.feature:4]: updated

File: src/test/java/ServiceNow/OA_Intake/Features/OAIntakeForm.feature (hunk 2 above)
Comment (alenaminsk) [reply src/test/java/ServiceNow/OA_Intake/Features/OAIntakeForm.feature:35]: updated

File: src/test/java/ServiceNow/OA_Intake/Features/OAIntakeForm.feature (hunk 3 above)
Comment (alenaminsk) [reply src/test/java/ServiceNow/OA_Intake/Features/OAIntakeForm.feature:37]: added

File: src/test/java/ServiceNow/OA_Intake/Features/OAIntakeForm.feature (hunk 4 above)
Comment (alenaminsk) [reply src/test/java/ServiceNow/OA_Intake/Features/OAIntakeForm.feature:87]: deleted

File: src/test/java/ServiceNow/OA_Intake/Steps/OAIntakeSteps.java
Comment (alenaminsk) [reply src/test/java/ServiceNow/OA_Intake/Steps/OAIntakeSteps.java:1]: kept Test ng for assertions

File: src/test/java/ServiceNow/OA_Intake/Steps/OAIntakeSteps.java (hunk 5 above)
Comment (alenaminsk) [reply src/test/java/ServiceNow/OA_Intake/Steps/OAIntakeSteps.java:293]: Kept testNG  for assertions

File: src/test/java/ServiceNow/OA_Intake/StepsImplementation/OAIntakeStepsImplementation.java
Comment (alenaminsk) [reply src/test/java/ServiceNow/OA_Intake/StepsImplementation/OAIntakeStepsImplementation.java:1]: updated

File: src/test/java/ServiceNow/OA_Intake/Features/OAIntakeForm.feature [hunk 7]
```diff
@@ -32,3 +32,52 @@ Feature: Create OA Intake form
...
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/OA_Intake/Features/OAIntakeForm.feature:83]: Please remove the empty lines 73-83

**Review by hamidnesar — CHANGES_REQUESTED** [review]

**Review by hamidnesar — APPROVED** [review]
Reviewed and approved!

**Review by sarwarahmed1 — APPROVED** [review]

**Review by matakevin — APPROVED** [review]
LGTM -- Thank you!

**Review by Mariachaudhry — APPROVED** [review]

**Pull request merged**